src/Components/Grid/Grid.scss

```diff
@@ -0,0 +1,102 @@
+.gridContainer {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    align-content: center;
+    justify-content: space-evenly;
+
+    .itemContainer {
+        // flex-grow: 1;
+        // flex-shrink: 1;
+        // flex-basis: 0;
+        flex-grow: 0;
+        flex-shrink: 1;
+        flex-basis: 1;
+        margin-bottom: 40px;
+        position: relative;
+        padding: 10px 10px;
+        align-self: flex-start;
+
+        &.width33 {
+            width: 33%;
+        }
+
+        &.width50 {
+            width: 50%;
+        }
+
+        &.width100 {
+            width: 100%;
+        }
+
+        .imgContainer {
+            position: relative;
+
+            .imgItem {
+                width: 100%;
+                height: 100%;
```

Comment:
![image](https://user-images.githubusercontent.com/26983421/162325373-5596f068-2d7e-474d-af8d-9460124273b0.png) Maybe height can be relative to a smaller container, in my opinion the img size is too height